### DISCUSSION TITLE
Manipulation of splines and traces

### DISCUSSION THREAD

**Discussion by rafa-cc** [xournalpp/xournalpp#5573] [Ideas]
Something i'd appreciate a lot is the chance to edit the points and tangents of a spline once its drawn. I try to draw with the greatest symmetry, however, often needs some adjusments and its kinda slow start from 0.

**Comment by rolandlo** [comment]
Eventually editable splines will come to Xournal++, but it will require a change in the file format. Splines are currently just saved as sequence of points.
